### PR TITLE
Review readme

### PR DESCRIPTION
**Pull request by Tianqi-Dotes**
Review readme.md

### CODE REVIEW

**Comment by CalvinKirs** [comment]
@Segun-Ogundipe PTAL,Thx

File: README.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ dolphinscheduler-dist/target/apache-dolphinscheduler-incubating-${latest.release
 ### Thanks
 
 Dolphin Scheduler is based on a lot of excellent open-source projects, such as google guava, guice, grpc, netty, ali bonecp, quartz, and many open-source projects of Apache and so on.
-We would like to express our deep gratitude to all the open-source projects which contribute to making the dream of Dolphin Scheduler comes true. We hope that we are not only the beneficiaries of open-source, but also give back to the community. Besides, we expect the partners who have the same passion and conviction to open-source will join in and contribute to the open-source community!
+We would like to express our deep gratitude to all the open-source projects assist Dolphin Scheduler. We hope that we are not only the beneficiaries of open-source, but also give back to the community. Besides, we expect the partners who have the same passion and conviction to open-source could join in and contribute to the open-source community!
```
Comment (Segun-Ogundipe) [inline README.md:84]: `We would like to express our deep gratitude to all the open-source projects assist Dolphin Scheduler` should be `We would like to express our deep gratitude to all the open-source projects used in Dolphin Scheduler`

Comment (Segun-Ogundipe) [reply README.md:84]: `we expect the partners who have the same passion and conviction to open-source could join in and contribute to the open-source community!` should be `we hope that partners who have the same enthusiasm and passion for open source could join in and contribute to the open-source community!`

**Review by Segun-Ogundipe — CHANGES_REQUESTED** [review]
Great job on this.

I left some comments here

**Review by dailidong — APPROVED** [review]
great job

File: README.md (hunk 1 above)
Comment (dailidong) [reply README.md]: I update the relate content, thanks for all your greate work

File: README.md [hunk 2]
```diff
@@ -15,38 +15,38 @@ Dolphin Scheduler Official Website
...
- - Support many task types: Shell, MR, Spark, SQL (MySQL, PostgreSQL, hive, spark SQL), Python, Sub_Process, Procedure, etc.
- - Support process scheduling, dependency scheduling, manual scheduling, manual pause/stop/recovery, support for failed retry/alarm, recovery from specified nodes, Kill task, etc.
- - Support the priority of process & task, task failover, and task timeout alarm or failure.
- - Support process global parameters and node custom parameter settings.
- - Support online upload/download of resource files, management, etc. Support online file creation and editing.
- - Support task log online viewing and scrolling, online download log, etc.
- - Implement cluster HA, decentralize Master cluster and Worker cluster through Zookeeper.
+ - Support various task types: Shell, MR, Spark, SQL (MySQL, PostgreSQL, hive, spark SQL), Python, Sub_Process, Procedure, etc.
+ - Support scheduling of workflows and dependencies, manual scheduling to pause/stop/recovery task, support failure task retry/alarm, recovery specified nodes from failure, kill task, etc.
```
[the remark this answers is not in the document]
Comment (Tianqi-Dotes) [reply README.md]: @Segun-Ogundipe 
hi Segun,
Deeply thanx for reviewing this passage.
Thanks
Yours sincerely
TIanqi

[the remark this answers is not in the document]
Comment (Segun-Ogundipe) [reply README.md]: You're welcome @Tianqi-Dotes

Regards

**Pull request merged**